Packages/Status/Sources/Status/Row/StatusRowViewModel.swift

```diff
@@ -30,15 +32,24 @@ public class StatusRowViewModel: ObservableObject {
 
   @Published var favoriters: [Account] = []
   @Published var rebloggers: [Account] = []
+
+  private let theme = Theme.shared
   
   var seen = false
 
   var filter: Filtered? {
     status.reblog?.filtered?.first ?? status.filtered?.first
   }
 
-  var shouldHighlightRow: Bool {
-    status.uiShouldHighlight != nil
+  var highlightRowColor: Color {
+
+    if status.visibility == .direct {
+      return theme.tintColor.opacity(0.15)
```

Comment:
Looking good

Reply:
I think it is subtle enough and indicates that this is a direct message instead of a highlight.